src/display/editor/editor.js

```diff
@@ -224,7 +224,7 @@ class AnnotationEditor {
     this.div.setAttribute("data-editor-rotation", (360 - this.rotation) % 360);
     this.div.className = this.name;
     this.div.setAttribute("id", this.id);
-    this.div.tabIndex = 0;
```

Comment:
I'm not sure *why* this was changed, since there ought to be no practical difference?

Reply:
I can't explain why.
I tested the patch on laptop with a touch screen and it was almost impossible to create a text editor in just tapping on the screen.
My feeling was that the div stole the focus from the nested editorDiv and then removed the editor.
So I just removed the tabindex and magically it worked.

Reply:
> So I just removed the tabindex and magically it worked.

But it's still being set, but slightly differently now; hence my question :-)

Reply:
Oh I thought it was just a removal in taking into account the context gh gives me here, and I was finally talking about this removal:
https://github.com/mozilla/pdf.js/pull/15188/files/b408d0e0abfd7d8c98e2befc82a7d4027e5d84e0#diff-8545c7e2bc39439352001789060a1bb9a42fc64e599c34322be45102ae3e7d08L221

Anyway you're right the change here is useless, but if iirc it was to be consistent with the line just above.
And I'm not sure the setAttribute on the above line is useful either.